pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -101,9 +112,10 @@ findDependency() {
 autoPatchelfFile() {
     local dep rpath="" toPatch="$1"
 
-    local interpreter="$(< "$NIX_CC/nix-support/dynamic-linker")"
+    local interpreter
+    interpreter="$(< "$NIX_CC/nix-support/dynamic-linker")"
```

Comment:
What's the reason for this? While it shouldn't make any difference it does make things less readable.